css/civicrm.css

```diff
@@ -887,6 +887,13 @@ input.crm-form-entityref {
   font-weight: bold;
 }
 
+/* dev/core#1039 Make contact details in popup on merge screen non bold */
+#crm-container tr.columnheader td .crm-summary-col-0,
+#crm-container tr.columnheader td .crm-summary-col-1 {
```

Comment:
or 
```css
#crm-container tr.columnheader td[class*="crm-summary-col-"] {
  font-size: 13px;
  font-weight: normal;
}
```

Reply:
O_o hadn't seen that one Monish

Reply:
same here .. its just a question arise on my mind when I saw those two lines "there have to be something to make it more generic .. then I found this https://stackoverflow.com/questions/10655221/multiple-matching-substring-selectors 

and then more of such occurrences in civicrm.css itself on grep 
```
css//crm-i.css:4:.ui-icon[class*=" fa-"],
css//crm-i.css:22:.ui-icon[class*=" fa-"] {
```